Debug App/Tests/Unit Tests/Data Models/IPay88Tests.swift

```diff
@@ -23,7 +23,8 @@ class IPay88Tests: XCTestCase {
         options: MerchantOptions(
             merchantId: "merchant-id",
             merchantAccountId: "merchant-account-id",
-            appId: "app-id"),
+            appId: "app-id",
+            extraMerchantData: nil),
```

Comment:
What do you think about adding a default value of nil for `extraMerchantData` to the initialiser so these callsites dont have to change?